src/style.css

```diff
@@ -41,19 +63,84 @@ h1 {
 .scores-con {
   border: 2px solid black;
   padding: 5px;
+  width: 300px;
+  max-width: 100%;
+  border-radius: 5px;
+  background-color: #6070ff;
```

Comment:
- [ ] [OPTIOANL] your app looks awesome I would love to suggest some enhancements to avoid large content scroll :

```CSS
.scores-con {
    border: 2px solid black;
    padding: 5px;
    width: 300px;
    max-width: 100%;
    border-radius: 5px;
    background-color: #6070ff;
   /*You can add max height and overflow-y scroll */
    max-height: 400px;
    overflow-y: scroll;
}
```

**_Results:_** 
![dcroll](https://user-images.githubusercontent.com/89282221/202449735-b68888b8-17f3-4532-9819-32f053d99cbf.gif)

Reply:
Thank you so much for this I appreciate